cpg_workflows/stages/large_cohort.py

```diff
@@ -68,19 +68,22 @@ def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
         # create these as empty lists instead of None, they have the same truthiness
         vds_paths: list[str] = []
         sg_ids_in_vds: list[str] = []
-        new_sg_gvcfs: list[str] | None = None
+        new_sg_gvcfs: list[str] = []
 
         if combiner_config.get('vds_analysis_ids', None) is not None:
             for vds_id in combiner_config['vds_analysis_ids']:
                 tmp_query_res, tmp_sg_ids_in_vds = self.get_vds_ids_output(vds_id)
                 vds_paths.append(tmp_query_res)
                 sg_ids_in_vds = sg_ids_in_vds + tmp_sg_ids_in_vds
 
-        if not config_retrieve(['combiner', 'merge_only_vds'], None):
+        if combiner_config.get('merge_only_vds', False) is not True:
```

Comment:
This is chaos, why not a Boolean? Lowercase unquoted true/false in the toml gets you a bool

Reply:
Yeah figured that out in testing just now, wasn't sure if that would happen